Makefile

```diff
@@ -229,12 +229,12 @@ build-local-ubuntu-all: build-local-linux  ## Build kusion & kcl tool chain for
 		./cmd/kcl-go
 
 	# Build kcl-openapi
-	# cd ./cmd/kcl-openapi && GOOS=linux GOARCH=amd64 CGO_ENABLED=0 \
-	# 	go build -o ${PWD}/_build/bundles/kusion-ubuntu/bin/kcl-openapi
+	cd ./cmd/kcl-openapi && GOOS=linux GOARCH=amd64 CGO_ENABLED=0 \
+		go build -o ${PWD}/_build/bundles/kusion-ubuntu/bin/kcl-openapi
 
 	# chmod +x
-	-chmod +x ./_build/bundles/kusion-linux/bin/kusion
```

Comment:
You can't rename linux to ubuntu, since script `build-local-linux` will only create the `kusion-linux` directory